src/main/java/domen/rideapp/domain/model/PricingConfig.java

```diff
@@ -0,0 +1,6 @@
+package domen.rideapp.domain.model;
+
+public record PricingConfig(double pricePerKm,
```

Comment:
Warto byłoby dodać informację co to za waluta, skoro już mamy tak rozbudowany config cenniku